automation/taskcluster/decisionlib.py

```diff
@@ -298,6 +324,8 @@ def build_worker_payload(self):
                 deindent("\n".join(self.scripts))
             ],
         }
+        if len(self.artifacts) > 0 and "chainOfTrust" not in self.features:
```

Comment:
This seems wrong.  There are some tasks that shouldn't have the CoT enabled because they're not trustworthy or don't want to trust their inputs.  Wouldn't this mean that any push to a PR, for example, will have CoT enabled?